packages/lab/src/typography/Text.css

```diff
@@ -9,13 +9,9 @@
 
 /* Main css class */
 .uitkText {
+  color: var(--uitk-text-primary-foreground);
   font-size: var(--uitk-text-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-lineHeight));
-}
-
-.uitkText-overflow {
-  height: 100%;
-  overflow: auto;
+  --uitkText-line-height: var(--uitk-text-lineHeight);
```

Comment:
The CSS variable naming doesn't align with our conventions.
 the variable `--uitkText-line-height` is an `API` variable - denoted by the mixed case --uitkComponentName prefix.
These should only ever be consumed within the component they belong to, not assigned values. They are for consumers or consuming components to supply overrides.

Reply:
but `--uitk-text-lineHeight` comes from characteristics. What else can I use in this situation?
@heswell @libertymayc